### PR TITLE
Update attestation service docs

### PR DESCRIPTION
**Pull request by eelanagaraj**
# Description
Updates attestation service docs (+ release process) to reflect current status.
Merge once the 1.5.0 release is out and double check the link.

Closes celo-org/celo-monorepo#9141

### CODE REVIEW

File: docs/validator-guide/attestation-service.md [hunk 1]
```diff
@@ -36,7 +37,10 @@ This guide steps you through setting up an Attestation Service:
 
 ## Recent releases
 
-- [Attestation Service v1.2.0](https://github.com/celo-org/celo-monorepo/releases/tag/attestation-service-v1.2.0) (latest production release)
+- [Attestation Service v1.5.0](https://github.com/celo-org/celo-monorepo/releases/tag/attestation-service-v1.5.0) (latest production release)
```
Comment (arthurgousset) [inline docs/validator-guide/attestation-service.md:40]: @eelanagaraj Thanks for including this! Just came across this while browsing the docs and remembered Cody's Slack message. Appreciate you updating this :)

**Review by arthurgousset — APPROVED** [review]
The release process notes and the error message overview for validators is super useful! Thanks for spending time on this 🙏 

("_LGTM_" 😄)

**Comment by arthurgousset** [comment]
Adding @critesjosh as reviewer (because I seem to remember he needs to approve any docs PRs).

**Comment by critesjosh** [comment]
I don't think you need me to approve as long as someone from @celo-org approves. Go ahead and merge--lmk if you can't.

**Comment by critesjosh** [comment]
Cool, looks like the build worked with @eelanagaraj merging.

**Pull request merged**